ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -125,6 +126,8 @@
   private final Thread commandRunnerThread;
   private final VersionCheckerAgent versionCheckerAgent;
 
+  private final ObjectMapper mapper;
```

Comment:
not sure this needs to be a field. How it was before works just as well.

Reply:
Doesn't have to be a field. It's only being used in `registerWebSocketEndpoints`. The intention of making it a field was to make reusing it easier. I moved it back to `registerWebSocketEndpoints` for now. We can make it a field later.